pg_analytics/Cargo.toml

```diff
@@ -30,6 +30,11 @@ deltalake = { git = "https://github.com/paradedb/delta-rs.git", branch = "feat/t
 thiserror = "1.0.56"
 indexmap = "2.2.2"
 once_cell = "1.19.0"
+datafusion-federation = { git = "https://github.com/paradedb/datafusion-federation.git", rev = "2d819a3" }
+datafusion-federation-sql = { git = "https://github.com/paradedb/datafusion-federation.git", rev = "2d819a3" }
+futures = "0.3.30"
+simple-logging = "2.0.2"
```

Comment:
Do you actually want to add these dependencies to this PR or were they just used for debugging purposees?

Reply:
good catch!